dds_web/api/project.py

```diff
@@ -592,7 +592,7 @@ def get(self):
         """Get private key from database."""
         # Verify project ID and access
         project = project_schemas.ProjectRequiredSchema().load(flask.request.args)
-
+        dds_web.utils.verify_project_user_key(project=project)
```

Comment:
remove the spaces here
```suggestion dds_web.utils.verify_project_user_key(project=project)
```

Reply:
Sorry, I didn't mean the line of code but the spaces which github is showing. Apparently there are no spaces, but it looks like there are
<img width="397" alt="Screenshot 2024-11-18 at 16 13 03" src="https://github.com/user-attachments/assets/9b5df846-e8cb-43ba-8ab7-7bd90814309b">